src/components/LoansByCategory/QuickFilters/QuickFilters.vue

```diff
@@ -179,13 +184,16 @@ export default {
 	components: {
 		KvSelect,
 		LocationSelector,
-		KvMaterialIcon
+		KvMaterialIcon,
+		FilterPills
 	},
 	data() {
 		return {
 			mdiFilterVariant,
+			mdiSort,
 			selectedCategory: 0,
 			selectedGender: '',
+			selectedGenders: ['all'], // TODO: consolidate to just this when UI changes apply to QF on lending home, too
```

Comment:
Not sure what this comment means. Please add a ticket and reference here with details on what this `TODO` entails.

Reply:
ya will do, just meant mostly that i don't know if it makes sense to have 2 gender states. The dropdown on lending home for gender should just use the array as well, there will just only ever be 1 element in the array at a time

Reply:
In that case, maybe update the comment to mention that the work on LH might mean removing the other gender state, I'm assuming `selectedGender`?

Reply:
Looks like this comment could still use some clarity, maybe mention that `selectedGender` could be removed, if that's the change needed here.